config/application.rb

```diff
@@ -67,9 +67,8 @@ class Application < Rails::Application
     # Globally handle Pundit::NotAuthorizedError by serving 404
     config.action_dispatch.rescue_responses["Pundit::NotAuthorizedError"] = :not_found
 
-    # After-initialize checker to add routes to reserved words
```

Comment:
I'm not only using a single `after_initialize` block so the comment doesn't make sense here anymore. It was a bit redundant with the one 2 lines below anyway.